File: app/assets/javascripts/models/car_engine.js
Vwkonfigurator.CarEngine = DS.Model.extend({
  _id: DS.attr('string'),
  token: DS.attr('string'),  
  primaryKey: "token",     
  name: DS.attr('string'),
  power: DS.attr('string'),
  transmision: DS.attr('string'),
  acceleration: DS.attr('string'),
  standart: DS.attr('string'),
  price: DS.attr('number'),
  description: DS.attr('string')
});

File: app/models/car_engine.rb
class CarEngine
  include Mongoid::Document
  include Mongoid::Token
  
  field :name, type: String
  field :power, type: String
  field :transmision, type: String
  field :acceleration, type: String
  field :standart, type: String
  field :price, type: String
  field :description, type: String

  token :length => 8, :contains => :fixed_numeric

end
